openmdao/docs/_utils/generate_sourcedocs.py

```diff
@@ -73,9 +73,11 @@ def generate_docs():
         'recorders',
         'solvers',
         'surrogate_models',
+        'surrogate_models.nn_interpolators',
```

Comment:
Surrogate entries not alphabetized. Supposes it really doesn't matter, but it looks like you keeping this in order.

Reply:
good point. easy fix. will do.

Reply:
(Done)